### PR TITLE
update and clean galital standalone chain specs and type

### PR DESCRIPTION
**Pull request by masterdubs**

### CODE REVIEW

File: packages/apps-config/src/endpoints/testing.ts [hunk 1]
```diff
@@ -139,8 +139,7 @@ export function createTesting (t: TFunction): LinkOption[] {
       }
     },
     {
-      info: 'rococoGalital',
-      paraId: 1230,
+      info: 'Galital',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:142]: We need a paraId to do the mapping.

Comment (masterdubs) [reply packages/apps-config/src/endpoints/testing.ts:142]: Thanks, fixed in next commit.

Comment (jacogr) [reply packages/apps-config/src/endpoints/testing.ts:142]: Still seems to be missing :) Just re-add the `paraId: 1230` line (assuming that is still the selected one)

Comment (masterdubs) [reply packages/apps-config/src/endpoints/testing.ts:142]: I can add it but it's an independent one, not a parachain version. Still need to add it ?

Comment (jacogr) [reply packages/apps-config/src/endpoints/testing.ts:142]: ^&*%$ My error, sorry. Dunno how that slipped in there initially, so the removal of it is 100% correct.

Ok, need some coffee it seems, not even half awake :)

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**